tests/parm/nems.configure.hafs_atm_ocn_wav.IN

```diff
@@ -79,27 +79,30 @@ WAV_attributes::
   Diagnostic = 0
   OverwriteSlice = false
   merge_import = .true.
+  mask_value_water = 1
+  mask_value_land = 0
 ::
 
 # Run Sequence #
 runSeq::
 @@[coupling_interval_sec]
-  ATM -> MED :remapMethod=redist
-  MED med_phases_post_atm
-  OCN -> MED :remapMethod=redist
-  MED med_phases_post_ocn
-  WAV -> MED :remapMethod=redist
-  MED med_phases_post_wav
   MED med_phases_prep_atm
```

Comment:
Is it required to start from "MED med_phases_prep_atm" first? What is the difference in the computation on CMEPS side between the new run sequence and the old one?

Reply:
Since the mediator itself calls the post phases in DataInitialize, there is no difference in the computation side, which is why baselines don't change by moving the post phases to the end where they belong. But if we wrote a mediator restart file, then there would be differences because the final import from a component wouldn't have yet take place when the mediator restart is written.

Any prep phase could go first, since they're mediator phases which just form the export state to the component. All components should have passed their updated values to CMEPS in the post phase at the end of the prior step through the run sequence.

It doesn't matter whether all prep phases come first, and then the all the redists from MED->component or whether the you group the components together (prep_atm, then med->atm, prep_ocn, med->ocn etc).

Reply:
Thanks for the explanation.